vscode/polar-lang/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "oso",
+  "publisher": "osohq",
+  "displayName": "oso",
+  "description": "Syntax highlighting for oso .polar files",
+  "icon": "oso_logo.png",
+  "version": "0.5.0",
```

Comment:
we have 0.1.0 in the changelog but 0.5.0 here

Reply:
Goes up every deploy and we did some test ones. Since we changed it to oso and I haven't deployed yet it'll go back to "0.1.0" when I do.

Reply:
Oh cool it auto-increments this?